src/cast_core/test/actors_test.cpp

```diff
@@ -12,16 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "test.h"
-
 #include <bsoncxx/builder/stream/document.hpp>
 #include <bsoncxx/json.hpp>
 #include <bsoncxx/types.hpp>
 
 #include <mongocxx/stdx.hpp>
 #include <mongocxx/uri.hpp>
 
-#include "MongoTestFixture.hpp"
+#include <testlib/MongoTestFixture.hpp>
+#include <testlib/helpers.hpp>
```

Comment:
do we want to be stricter about the namespaces lining up with the include dir? making `testlib` files define only things in the `testlib::` namespace? Seems "right" but also like it could get tedious.

Reply:
If we only have to fix up files in testlib it seems worth it. (2c)